recipes/libpdal/build.sh

```diff
@@ -53,7 +53,7 @@ if [ $(uname) == Darwin ]; then
 else
     COMP_CC=gcc
     COMP_CXX=g++
-    export LDFLAGS="${LDFLAGS} -lkea -lproj -lpoppler -lxerces-c-3.2 -ldap -lpng16 -lpcre -liconv -lhdf5 -lgeos-3.6.2 -licui18n -licuuc -licudata -lcom_err -lhdf5_hl -lhdf5_cpp -lminizip -luriparser -lkmlbase -lkmldom -lkmlengine -ldapclient -lssl -lssh2 -lcrypto -lmfhdf -ldf -ljpeg -ltiff"
+    export LDFLAGS="${LDFLAGS} -lkea -lproj -lpoppler -lxerces-c-3.2 -ldap -lpng16 -lpcre -liconv -lhdf5 -lgeos-3.6.2 -licui18n -licuuc -licudata -lcom_err -lhdf5_hl -lhdf5_cpp -lminizip -luriparser -lkmlbase -lkmlconvenience -lkmldom -lkmlengine -ldapclient -lssl -lssh2 -lcrypto -lmfhdf -ldf -ljpeg -ltiff -lpq"
 fi
```

Comment:
I had to do this when I compiled on my machine. I had the PR ready. 😄

Reply:
I was wondering if we should add explicit dependencies on these libraries we're linking.  Right now we're getting many of them because gdal or something else brings them in.  What do you think?

Reply:
If we already depend on gdal and that brings them in, we are fine. We don't want to double specify these library and then keep them in sync in both places.